src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5481,7 +5560,10 @@ export class StudyViewPageStore {
                 STUDY_VIEW_CONFIG.layout.dimensions[ChartTypeEnum.SURVIVAL]
             );
             // check priority for survival plots, hide the plot if priority is 0
-            if (this.chartMetaSet[key].priority !== 0) {
+            if (
+                key in this.chartMetaSet &&
```

Comment:
Added this check due to this error:
![image](https://user-images.githubusercontent.com/33106214/105233196-848b9080-5b37-11eb-95d4-0e5a659244eb.png)
![image](https://user-images.githubusercontent.com/33106214/105233227-8bb29e80-5b37-11eb-9257-e9d3190a415e.png)

Reply:
weird. how come this error doesn't happen in mast code